lib/components/primitive-components/Group/Group.ts

```diff
@@ -469,6 +469,15 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
    * or if using a "fullview" or "rip and replace" autorouting mode
    */
   _shouldUseTraceByTraceRouting(): boolean {
+    // Skip if the parent subcircuit is performing async autorouting
+    if (
+      this.parent &&
+      this.parent instanceof Group &&
+      this.parent._shouldRouteAsync()
+    ) {
+      return false
+    }
```

Comment:
not the right behavior, this is the correct behavior:

- The props define what the subcircuit should do FIRST
- If the props don't specify what you should do, THEN you look at your subcircuit, _not your parent_

Reply:
you can't do trace-by-trace autorouting until your children have finished autorouting, but that's outside the scope of this function and may already be implemented